### PR TITLE
Feature/general 3rd milestone

### PR DESCRIPTION
**Pull request by AmineDerbal**
In this milestone :

- added the contact section to my portfolio
- my coding partner contributed by creating the form and the field button
- implemented flexbox to the section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -277,5 +277,69 @@ <h3>Skills</h3>
...
+        id="contact-form"
+        action="https://formspree.io/f/xayznown"
+        method="POST"
+      >
+        <ul id="form-list">
+          <li>
+            <input
+              type="text"
+              name=""
+              id=""
```
Comment (Olamarx) [inline index.html]: - [ ] The form is not getting submitted successfully. I would advise you to fill out the empty attributes as the name attribute is the issue here

image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @AmineDerbal, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️ 
- Readme file ✔️ 
- Form tag ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -293,21 +293,32 @@ <h3>Skills</h3>
...
-Enter text here</textarea
-            >
+            <textarea
+              name="message"
+              id="message"
+              cols="30"
+              rows="10"
+              placeholder="Enter text here"
+              required
+            ></textarea>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ Optional ] As stated the project requirement Kindly make sure that Message field accepts up to 500 chars.
 
Capture

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @AmineDerbal 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Pull request merged**